src/logging/drain.rs

```diff
@@ -0,0 +1,59 @@
+use anyhow::Result;
+use slog::{OwnedKVList, Record};
+
+use super::event;
+
+/// A logging drain designed to integrate with [`slog::Drain`]
+///
+/// The drain can be easily added to a regular [`slog::Logger`]:
+///
+/// ```rust
+/// use kubewarden_policy_sdk::logging;
+/// use slog::{Drain, Logger, o};
+///
+/// let drain = logging::KubewardenDrain::new().fuse();
+/// let log = Logger::root(drain, o!("logger_key1" => "logger_value1"));
```

Comment:
I think this two lines could become:

```suggestion
/// let log = Logger::root(logging::KubewardenDrain::default(), o!("logger_key1" => "logger_value1"));
```

Reply:
done